modules/apps/object/object-api/src/main/java/com/liferay/object/entry/permission/util/ObjectEntryPermissionUtil.java

```diff
@@ -40,16 +40,15 @@
 public class ObjectEntryPermissionUtil {
 
 	public static void checkAccountEntryPermission(
-			AccountEntryLocalService accountEntryLocalService, String actionId,
-			ObjectDefinition objectDefinition, ObjectEntry objectEntry,
-			ObjectFieldLocalService objectFieldLocalService, long userId)
+			String actionId, ObjectDefinition objectDefinition,
+			ObjectEntry objectEntry, long userId)
 		throws PortalException {
 
 		if (!objectDefinition.isAccountEntryRestricted()) {
 			return;
 		}
 
-		ObjectField objectField = objectFieldLocalService.getObjectField(
+		ObjectField objectField = ObjectFieldLocalServiceUtil.getObjectField(
```

Comment:
Sure @gabrielwas!

I did this way because I've seen this [example](https://github.com/liferay/liferay-portal/blob/6794b295d34ead99c5b517dc4f647d7e8ffa82d7/modules/apps/bookmarks/bookmarks-web/src/main/java/com/liferay/bookmarks/web/internal/portlet/util/BookmarksUtil.java#L149) and I believed that will be more simply

Thank you!

Reply:
Done!